.gitignore

```diff
@@ -1,3 +1,3 @@
 node_modules/
 npm-debug.*
-coverage/
+coverage/
```

Comment:
A new line at the end of the file is missing. This file must be exactly equal to the old one.

Reply:
ups  changing right now